library/src/main/java/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -12,6 +17,8 @@ import com.chuckerteam.chucker.internal.ui.MainActivity
  */
 public object Chucker {
 
+    private const val SHORTCUT_ID = "shortcutId"
```

Comment:
Make it more specific to Chucker. Like `chuckerShortcutId` or something. I'd prefer to avoid situation where someone has defined such a value as well and our shortcut is not added.

Reply:
Changed